interp/handler_test.go

```diff
@@ -21,29 +21,29 @@ import (
 	"mvdan.cc/sh/v3/syntax"
 )
 
-func blacklistBuiltinExec(name string) ExecHandlerFunc {
+func blocklistBuiltinExec(name string) ExecHandlerFunc {
```

Comment:
"blacklist" => "blocklist" 👍👍👍

Reply:
Indeed on purpose, I just forgot to mention it in the commit message :) done.